adobe/spectrum-web-components
# fix(action-menu): dispatch scroll event

## Description
`@scroll` event is attached to the `sp-menu` in the Picker's render tree. 




## Related issue(s)



-  https://jira.corp.adobe.com/browse/SWC-455

## Motivation and context
Switched one of our components from `sp-menu` to `sp-action-menu` in order to leverage the mobile UI built in with `sp-action-menu`.
However, when we made the switch, we noticed that `@scroll` stopped working as expected,


## How has this been tested?




    1. Go [here](https://rajdeepc-swc-455--spectrum-web-components.netlify.app/storybook/?path=/story/action-menu--with-scroll-event)
    2. open console
    3. Scroll the list either by `mousewheel` or `up` and `down` keys.
    4. check the @scroll event is getting fired and console logging
  


## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
